deps/wazuh_testing/wazuh_testing/modules/logcollector/event_monitor.py

```diff
@@ -132,3 +133,43 @@ def check_ignore_restrict_message_not_found(message, regex, tag, prefix):
     with pytest.raises(TimeoutError):
         log_found = check_ignore_restrict_message(message=message, regex=regex, tag=tag, prefix=prefix)
     assert log_found is False, ERR_MSG_UNEXPECTED_IGNORE_EVENT
+
+
+def check_wildcard_pattern_expanded(file_path, location_regex, prefix, error_message=None, file_monitor=None,
+                                    timeout=T_10, escape=False):
+    """Create a callback to detect "New file that matches the '{file_path}' pattern: '(.*)'" line.
+
+    Args:
+        file_path (str): file path that is being monitored
+        location_regex (str): path configured in location tag
+        prefix (str): Daemon that generates the error log.
+        error_message (str): Error message.
+        file_monitor (FileMonitor): Log monitor.
+        timeout (int): Timeout to check the log.
+        escape (bool): Flag to escape special characters in the pattern.
+
+    Returns: True if the expected message has been found, False otherwise.
+    """
+    callback_msg = f".*New file that matches the '{location_regex}' pattern: '{file_path}'"
+
+    return check_logcollector_event(file_monitor=file_monitor, timeout=timeout, callback=callback_msg,
+                                    error_message=error_message, prefix=prefix, escape=escape)
+
+
+def check_wildcard_pattern_no_match(regex, prefix, error_message=None, file_monitor=None, timeout=T_10, escape=False):
```

Comment:
```suggestion
def check_win_wildcard_pattern_no_match(regex, prefix, error_message=None, file_monitor=None, timeout=T_10, escape=False):
```

It seems that this log only applies to Windows.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3906/commits/ffca56f7b9804b4ef1df2c7ffdd10987414e6e48